features/openhab-addons-external/src/main/resources/conf/dynamodb.cfg

```diff
@@ -47,5 +47,12 @@
 # write capacity for the created tables
 #writeCapacityUnits=1
 
+# if auto expiration from ddb is enabled (ttl) [ defaults to false ]
```

Comment:
Use DynamoDB instead previously unmentioned abbreviation `ddb`. Since other parameters do not have `[ defaults to XX ]` neither should this one. The default is there in the comments.